apps/graph/values/values_controller.cpp

```diff
@@ -230,23 +213,38 @@ ViewController * ValuesController::functionParameterController() {
   return &m_functionParameterController;
 }
 
-double ValuesController::evaluationOfAbscissaAtColumn(double abscissa, int columnIndex) {
+void ValuesController::printEvaluationOfAbscissaAtColumn(double abscissa, int columnIndex, char * buffer, const int bufferSize) {
   bool isDerivative = false;
+  double evaluationX = NAN;
+  double evaluationY = NAN;
   Ion::Storage::Record record = recordAtColumn(columnIndex, &isDerivative);
   Shared::ExpiringPointer<ContinuousFunction> function = functionStore()->modelForRecord(record);
   Poincare::Context * context = textFieldDelegateApp()->localContext();
+  bool isParametric = function->plotType() == ContinuousFunction::PlotType::Parametric;
   if (isDerivative) {
-    return function->approximateDerivative(abscissa, context);
+    evaluationX = function->approximateDerivative(abscissa, context);
+  } else {
+    Poincare::Coordinate2D<double> eval = function->evaluate2DAtParameter(abscissa, context);
+    evaluationX = eval.x2();
```

Comment:
Change the name of these variables?
evaluationX -> evaluationX2 (or evaluationY)
evaluationY -> evaluationX1 (or evaluationX)